app/styles/play/level/hints-view.sass

```diff
@@ -54,3 +55,21 @@
       .ace_cursor-layer
         .ace_cursor
           display: none
+
+  .chatbot-help
+    text-align: center
+    border-top: 1px solid white
+    padding-top: 10px
+    margin-top: 10px
+
+    img.avatar
+      width: 40px
+      margin-right: 12px
+
+    button
+      margin-left: 6px
```

Comment:
we dont have an convention of sizes but I generally prefer margin and padding to be multiple of 5s and font-sizes to be multiple of 2